archiver_test.go

```diff
@@ -74,7 +74,7 @@ func symmetricTest(t *testing.T, name string, ar Archiver) {
 
 		if info.IsDir() {
 			// stat dir instead of read file
-			_, err := os.Stat(origPath)
+			_, err = os.Stat(origPath)
```

Comment:
It's great being a software developer, where a single colon is a starting point for a conversation. 👍